src/examples/deployment.yml

```diff
@@ -16,25 +16,8 @@ usage:
       steps:
         - checkout
         - kube-orb/install-kubectl
-        - run:
-            name: Start minikube
-            command: |
-              curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
-                && chmod +x minikube
-              sudo cp minikube /usr/local/bin && rm minikube
-              sudo -E minikube start --vm-driver=none --cpus 2 --memory 2048
         - kube-orb/create-or-update-resource:
             resource-file-path: "tests/nginx-deployment/deployment.yaml"
             resource-name: "deployment/nginx-deployment"
             get-rollout-status: true
             show-kubectl-command: true
-        - kube-orb/update-container-image:
```

Comment:
Could you move the `kube-orb/update-container-image` step to `src/examples/deployment-update.yml` (which seems to be an empty file at the moment) ?

Reply:
I thought I'd done this already, sorry. Updated in https://github.com/CircleCI-Public/kubernetes-orb/pull/26/commits/9fafd60156a64043daeee4bea83d351893fe3e0e